.github/workflows/actions.yml

```diff
@@ -176,6 +176,11 @@ jobs:
       run: |
         kubectl create ns buoyant-cloud
 
+    - name: Build and load agent image
+      run: |
+        docker build  -f agent/Dockerfile . -t bcloud-agent:ci-run
+        kind load docker-image bcloud-agent:ci-run --name=chart-testing
```

Comment:
can you add a `Cache Docker layers` step, similar to the `Docker Build and Push` job? i think all we need is that one step, i don't think we need the `Move cache` step, since this doesn't use `buildkit`.

### Before
<img width="590" alt="Screen Shot 2022-02-18 at 3 17 45 AM" src="https://user-images.githubusercontent.com/236915/154672862-c5860970-27f4-4ac7-93c2-2ba32cd36400.png">

### After
<img width="600" alt="Screen Shot 2022-02-18 at 3 14 25 AM" src="https://user-images.githubusercontent.com/236915/154672638-1ece3a55-05b4-4cf9-a8d5-6055466cbd7c.png">